roles/common/tasks/main.yml

```diff
@@ -1,11 +1,26 @@
 ---
-- debug:
-    msg: "{{ansible_architecture}}"
 
-- name: Update apt cache
+- name: Install network tools
   apt:
-    update-cache: yes
```

Comment:
Updating the cache is not needed?

Reply:
It's been working without it, but maybe it's important for the long term